catalog/actions.go

```diff
@@ -0,0 +1,56 @@
+package catalog
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/treeverse/lakefs/graveler"
+
+	"github.com/treeverse/lakefs/block"
+)
+
+type actionsSource struct {
+	catalog *EntryCatalog
+	adapter block.Adapter
+
+	repositoryID graveler.RepositoryID
+	repository   graveler.Repository
+	ref          graveler.Ref
+}
+
+const actionsPrefix = "_lakefs_actions/"
+
+func (as *actionsSource) List(ctx context.Context) ([]string, error) {
```

Comment:
This implementation can be an issue - as we do not return the real name here.

Reply:
Done.